akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -366,7 +366,7 @@ object ShardCoordinator {
     regions.foreach(_ ! BeginHandOff(shard))
     var remaining = regions
 
-    import context.dispatcher
+    private implicit val ec: ExecutionContext = context.dispatcher
```

Comment:
@johanandren you said that we could't have done mistake in internals. Isn't this an example of that it could happen if scheduler is triggered at the same time as the actor is stopping (setting the null)? Small chance but possible, I think.

Reply:
No, because the implicit is captured on the next line, in the constructor and it not referenced after that.